op-program/client/l2/test/stub_oracle.go

```diff
@@ -35,7 +34,6 @@ func NewStubOracle(t *testing.T) (*StubBlockOracle, *StubStateOracle) {
 	blockOracle := StubBlockOracle{
 		t:                t,
 		Blocks:           make(map[common.Hash]*gethTypes.Block),
-		BlockData:        make(map[common.Hash]*gethTypes.Block),
```

Comment:
Every case this is used it was adding the same blocks to Blocks and BlockData anyway. The action tests now ensure re-generation really is triggered by calling BlockDataByHash when required.